src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -60,16 +60,13 @@ public static int IndexOfAny(ref byte searchSpace, int searchSpaceLength, ref by
             if (valueLength == 0)
                 return 0;  // A zero-length sequence is always treated as "found" at the start of the search space.
 
-            int index = -1;
+            int index = int.MaxValue;
```

Comment:
You should be able to start with -1 here and avoid the normalization at the end.

Reply:
Great -- the "uint-trick" opens the door for this 😄